examples/virtio/blk_driver_vmm.c

```diff
@@ -44,25 +48,19 @@ extern char _guest_initrd_image_end[];
 /* Microkit will set this variable to the start of the guest RAM memory region. */
 uintptr_t guest_ram_vaddr;
 
-uintptr_t serial_rx_free;
-uintptr_t serial_rx_used;
-uintptr_t serial_tx_free;
-uintptr_t serial_tx_used;
-
-uintptr_t serial_rx_data;
-uintptr_t serial_tx_data;
-
-ring_handle_t serial_rx_ring;
-ring_handle_t serial_tx_ring;
-
-#define SERIAL_MUX_TX_CH 1
-#define SERIAL_MUX_RX_CH 2
+/* sDDF block */
+#if defined(BOARD_odroidc4)
```

Comment:
Does the driver VM work on QEMU? I would have expected some IRQs for QEMU to also be passed through.

Reply:
The qemu example uses a ramdisk that's packaged into userspace, I had thought passing through users own local disk would not be the best idea. But I believe we can supply a virtual disk to qemu that wouldn't write into user's disk directly. Will get around to that when I can